frontend/src/components/AccountReceiveTable.vue

```diff
@@ -100,7 +100,7 @@
                         ? formatNameOrAddress(props.row.formattedFrom)
                         : formatNameOrAddress(props.row.from)
                     }}</span>
-                    <loading-spinner v-if="!props.row.formattedFrom" size="1rem" customClass="q-ml-sm" />
+                    <loading-spinner v-if="!props.row.formattedFrom" size="16px" customClass="q-ml-sm" />
```

Comment:
Hm do you know why rem was a problem? It seems odd because the quasar docs [say](https://quasar.dev/vue-components/spinners) that rem is supported. Rem is nicer than px because px is absolute and rem is relative, so if the user changes the browser font size, rem units scale accordingly but pixel units don't

Reply:
Ahh didn't realize rem had those benefits. Did some testings, those warnings are only showing up when using Firefox. Chromium-based browsers seem to be fine with rem.

Reply:
Using `em` gets rid of those errors. Should I use that instead or close this PR entirely.

Reply:
Are things not rendering correctly in firefox because of the rem unit, or do they render correctly and we just want to get rid of the warning? 

Either way, em and rem are both relative so using em is just as good. But they are relative to a different base, so you might need to change the values (or not, if both bases are the same value)

Reply:
they are rendering correctly but wanted to get rid of the warnings.

Reply:
Updated to use em here: https://github.com/ScopeLift/umbra-protocol/pull/592/commits/8cd7ef181f8438eb2ab20a9afc5fc05bf927479d

Reply:
This might be what is going on here. Although, I am surprised Firefox doesn't support SVG 2 yet

https://stackoverflow.com/questions/46336500/svg-text-layout-rendering-differently-in-chrome-firefox-edge/46337386#46337386